### PR TITLE
在ray_ex.lean中添加了六个需要证明的小定理

### PR DESCRIPTION
**Pull request by XintaoYu**

### CODE REVIEW

File: EuclideanGeometry/Foundation/Axiom/Linear/Ray_ex.lean [hunk 1]
```diff
@@ -208,6 +208,30 @@ theorem lies_on_pt_toDir_of_pt_lies_on_rev {A B : P} {ray : Ray P} (hA : A LiesO
   simp only
   rw[add_smul,←vec_sub_vec ray.source,ha,hb,sub_neg_eq_add]
 
+-- Given two rays $ray_1$ and $ray_2$ with same direction, if the source of $ray_1$ lies on $ray_2$, then the source of $ray_2$ lies on the reverse of $ray_1$
+
+theorem lies_on_iff_lies_on_rev_of_same_todir {ray₁ ray₂ : Ray P} (h : ray₁.toDir = ray₂.toDir) : ray₁.source LiesOn ray₂ ↔ ray₂.source LiesOn ray₁.reverse := sorry
```
Comment (jjdishere) [inline EuclideanGeometry/Foundation/Axiom/Linear/Ray_ex.lean:213]: `same` should be `eq`

**Review by jjdishere — CHANGES_REQUESTED** [review]

File: EuclideanGeometry/Foundation/Axiom/Linear/Ray_ex.lean (hunk 1 above)
Comment (jjdishere) [reply EuclideanGeometry/Foundation/Axiom/Linear/Ray_ex.lean:213]: also occurs in the next theorem

**Comment by jjdishere** [comment]
If you finished your change, please leave a comment here to ask for a new review. Otherwise I won't start to look at your changes. Thanks!

File: EuclideanGeometry/Foundation/Axiom/Linear/Ray_ex.lean (hunk 1 above)
Comment (XintaoYu) [reply EuclideanGeometry/Foundation/Axiom/Linear/Ray_ex.lean:219]: 收到，已修改

**Pull request closed without merging**